### ISSUE TITLE
Source map pollution

### ISSUE THREAD

**Issue by ryanleecode** [pfgray/ts-adt#14]
```
⚠️  Could not load source file "../src/ADT.ts" in source map of "../node_modules/ts-adt/ADT.js".
⚠️  Could not load source file "../src/MakeADT.ts" in source map of "../node_modules/ts-adt/MakeADT.js".
```

![image](https://user-images.githubusercontent.com/21375952/101426433-f92da700-38ca-11eb-9791-e641529b27c2.png)

**Comment by pfgray** [comment]
Oh, I think this was cause from me using some release code from fp-ts... @ryanleecode , Could you share what tool you're using so I can make sure it works with that?

**Comment by ryanleecode** [comment]
Was just using [Parcel](https://parceljs.org/getting_started.html)

**Comment by pfgray** [comment]
@ryanleecode I think the issue should be resolved w/ 2.0.2 can you confirm?

**Comment by ryanleecode** [comment]
Unchanged
